pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -532,6 +532,45 @@ func Test_EventListenerValidate(t *testing.T) {
 				}},
 			},
 		},
+	}, {
+		name: "Valid EventListener with node affinity",
+		el: &v1alpha1.EventListener{
+			ObjectMeta: metav1.ObjectMeta{
```

Comment:
do we need to add this in both v1alpha1 and v1beta1?

Reply:
Yes you are right we need to add for both because this is not something new for beta

Reply:
Also we don't specify supported for alpha and beta here https://github.com/tektoncd/triggers/blob/main/docs/eventlisteners.md#specifying-resources that why added in both in order to not to confuse

Reply:
> Do you think new supported fields can be added only for beta not for alpha 🤔

Yeah, that's how pipelines did it. They essentially froze v1alpha1 and only added new features to v1beta1 for simplicity.